### PR TITLE
Store engine hours in milliseconds for the rest of protocols

### PR DESCRIPTION
**Pull request by Abyss777**
Next step for #3532 

Assumed that most of protocol reports in hours.

### CODE REVIEW

File: src/org/traccar/protocol/AplicomProtocolDecoder.java [hunk 1]
```diff
@@ -549,7 +549,7 @@ private void decodeF(Position position, ChannelBuffer buf, int selector) {
         }
 
         if ((selector & 0x0020) != 0) {
-            position.set(Position.KEY_HOURS, buf.readUnsignedInt());
+            position.set(Position.KEY_HOURS, buf.readUnsignedInt() * 3600000);
```
Comment (tananaev) [inline src/org/traccar/protocol/AplicomProtocolDecoder.java]: Do you think we should add this to unit converter class?

**Comment by Abyss777** [comment]
Function name rather long, what about `msFromHours` ? Or it is OK?

**Comment by tananaev** [comment]
Yeah, I like `msFromHours`.

**Comment by Abyss777** [comment]
Renamed

File: src/org/traccar/protocol/MxtProtocolDecoder.java [hunk 2]
```diff
@@ -145,7 +145,7 @@ protected Object decode(
             }
 
             if (BitUtil.check(infoGroups, 4)) {
-                position.set(Position.KEY_HOURS, buf.readUnsignedInt());
+                position.set(Position.KEY_HOURS, buf.readUnsignedInt() * 60000);
```
Comment (tananaev) [inline src/org/traccar/protocol/MxtProtocolDecoder.java]: Sorry, haven't noticed it before. I think all of the conversions should be using `UnitsConverter`.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**